app/views/questions/show.html.erb

```diff
@@ -1,17 +1,18 @@
 <% content_for :title, @question.title.truncate(50) %>
 
 <%= render 'posts/expanded', post: @question %>
-<span class="clearfix"></span><br/>
-<h4><%= pluralize(@question.answers.where(deleted: false).count, "answer") %></h4>
 
-<%= tabs do %>
-  <% tab 'Score', query_url(sort: 'score'), is_active: active_search?('score') %>
-  <% tab 'Age', query_url(sort: 'age'), is_active: active_search?('created_at') %>
-<% end %>
+<h2><%= pluralize(@question.answers.where(deleted: false).count, "answer") %></h2>
+
+<div class="button-list is-gutterless has-float-right">
+  <a href="<%= query_url(sort: 'score') %>" class="button is-muted is-outlined <%= active_search?('score') ? 'is-active' : '' %>">Score</a>
+  <a href="<%= query_url(sort: 'age') %>" class="button is-muted is-outlined <%= active_search?('created_at') ? 'is-active' : '' %>">Active</a>
+</div>
```

Comment:
Here, let's change the `tabs` helper instead, so that the code in the view here can remain the same. It's defined in [tabs_helper.rb](https://github.com/codidact/qpixel/blob/master/app/helpers/tabs_helper.rb), and you can change the element classes in there.

Reply:
Not sure about that. Co-Design has these button lists (which are intended for filtering/sorting) and it has real tabs (such as tabs on a profile page).

Reply:
Tabs feel more intuitively correct to me for this use, because clicking on them results in a full navigation to a new page, not just a client-side re-sort. Either way, though - whichever we pick, the code for it can go in `tabs_helper` and we can call `tabs` in the view.